datastore/integration-test/response/items_single_id.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Can we use the id that is in `items_area_with_one_parameter_name.json` so we can drop this file?